common/src/main/java/com/wynntils/services/mapdata/providers/builtin/CategoriesProvider.java

```diff
@@ -122,10 +123,10 @@ public MapVisibility getLabelVisibility() {
     }
 
     private static final class FoundChestCategory implements MapCategory {
-        private static final MapVisibility TIER_1_VISIBILITY = new MapVisibility(57, 100, 6);
-        private static final MapVisibility TIER_2_VISIBILITY = new MapVisibility(57, 100, 6);
-        private static final MapVisibility TIER_3_VISIBILITY = new MapVisibility(30, 100, 6);
-        private static final MapVisibility TIER_4_VISIBILITY = new MapVisibility(30, 100, 6);
+        private static final MapVisibility TIER_1_VISIBILITY = new FullMapVisibility(57, 100, 6);
```

Comment:
My idea here is that we should only need to set the min level to 57, and inherit the rest. Maybe we can do that by a builder pattern? Like `MapVisibility.builder().setMin(57).build()`?

(and similarly for the rest of these default values -- only the ones that are changed wrt to the default should be actually set)

Reply:
We can make it DerivedVisibilty.withMin(28) and it'll work.

Reply:
Sounds good, let's do that.